sys/net/if_bridge.c

```diff
@@ -3728,6 +3747,7 @@ bridge_ip_checkbasic(struct mbuf **mp)
 	*mp = m;
 	return (-1);
 }
+#endif
```

Comment:
The `#ifdef #endif` block is long, and I think comment `/* INET */` after `#endif` will improve readability.